google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/impl/AppEngineCommunityWebIntegration.java

```diff
@@ -82,18 +81,15 @@ public void setupJpaSupport(@NotNull Module module, @NotNull VirtualFile persist
   }
 
   @Override
-  public void setupRunConfiguration(@NotNull AppEngineSdk sdk,
-      @Nullable Artifact artifact,
-      @NotNull Project project) {
+  public void setupRunConfiguration(@Nullable Artifact artifact, @NotNull Project project) {
```

Comment:
why is this class called ..Community..?  (Lots of questions from ignorance here. sorry)